infrastructure/annotations-api/src/dynamodb.ts

```diff
@@ -49,6 +49,16 @@ export class DynamoDB extends Construct {
           },
         ],
       },
+      lifecycle: {
+        ignoreChanges: [
+          // Bug in terraform with DynamoDB and global secondary indexes
```

Comment:
😮‍💨

Reply:
from 2017? yikes

Reply:
Can you add this link to the PR description so that it gets tagged?